frontend/layouts/default.vue

```diff
@@ -156,6 +162,42 @@ export default Vue.extend({
                 this.$store.dispatch('network/setup');
             }
         },
+        electronUpdateVersion(newValue) {
+            if (newValue) {
+                notification.info({
+                    message: 'Update available',
+                    description: {
+                        tag: 'div',
+                        children: [
+                            {
+                                tag: 'span',
+                                text: `A new version (v${newValue}) is available for download at `,
+                                isRootInsert: false,
+                                isComment: false,
+                            },
+                            {
+                                tag: 'a',
+                                text: 'https://github.com/sidestream-tech/unified-auctions-ui/releases/latest',
```

Comment:
Please do not hardcode this value in the frontend, but take it from the electron call (as you already have it in electron/package.json)